x-pack/plugins/security_solution/public/common/components/formatted_date/index.tsx

```diff
@@ -22,6 +22,24 @@ export const PreferenceFormattedDate = React.memo<{ dateFormat?: string; value:
 
 PreferenceFormattedDate.displayName = 'PreferenceFormattedDate';
 
+export const PreferenceFormattedDateFromPrimitive = ({
```

Comment:
Since this is a new component under `.../public/common`, I would suggest adding some jsdocs here

Also - wondering why this was needed - `<FormattedDate/>` (below) did not handle this use case?